packages/ui/src/components/va-collapse/VaCollapse.vue

```diff
@@ -224,7 +224,7 @@ export default defineComponent({
 
   &__body-wrapper {
     transition: var(--va-collapse-body-wrapper-transition);
-    overflow: hidden;
+    overflow: scroll;
```

Comment:
```suggestion
    overflow: auto;
```

> My guess is that collapse should always have scroll in this case

I think it meant that it must have scroll in case of overflow.

Reply:
That makes sense.. Applied the suggestion :)